templates/api/dataset_loading/index.md

```diff
@@ -25,29 +30,29 @@ You may also have a validation data folder `validation_data/` structured in the
 You could simply do:
 
 ```python
-from tensorflow import keras
-from tensorflow.keras.preprocessing import image_dataset_from_directory
+import tensorflow as tf
 
-train_ds = image_dataset_from_directory(
+train_ds = tf.keras.utils.image_dataset_from_directory(
```

Comment:
Generally speaking, prefer importing `keras` and using that as the top-level module -- there's not telling when we'll start distributing Keras as a standalone multi-backend package again :)